charts/morpheus-ai-engine/.frigate

```diff
@@ -0,0 +1,21 @@
+{% extends "rst.jinja2" %}
+
```

Comment:
why is morpheus in this release?

Reply:
These are the charts they aren't deployed automatically just bringing the charts local so I don't have to go looking for them (same with jupyterhub) also for testing with terraform etc